internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -147,7 +147,11 @@ Resources:
               OriginAccessIdentity: ''
         {{- if .StaticSiteAlias}}
         ViewerCertificate:
+        {{- if .StaticSiteCert}}
+          AcmCertificateArn: {{.StaticSiteCert}}
```

Comment:
Does this mean we can not have multiple imported certificates for static site  as we have to provide a string for 
`AcmCertificateArn`?

Reply:
Yep multiple certs is not supported by CF.